pwa/src/components/endpoints/endpointForm.tsx

```diff
@@ -5,140 +5,116 @@ import {
   Alert,
   SelectInputComponent,
   Card,
-  Accordion
 } from "@conductionnl/nl-design-system/lib";
-import {isLoggedIn} from "../../services/auth";
 import {Link} from "gatsby";
 import {navigate} from "gatsby-link";
 import {
   checkValues,
   removeEmptyObjectValues,
 } from "../utility/inputHandler";
 import FlashMessage from 'react-flash-message';
-import LoadingOverlay from "../loadingOverlay/loadingOverlay";
+import APIService from "../../apiService/apiService";
+import APIContext from "../../apiService/apiContext";
+import LoadingOverlay from '../loadingOverlay/loadingOverlay'
 
 interface EndpointFormProps {
   id: string,
 }
 
 export const EndpointForm: React.FC<EndpointFormProps> = ({id}) => {
-
-  const [context, setContext] = React.useState(null);
+  const [alert, setAlert] = React.useState<Record<string, string>>(null);
   const [endpoint, setEndpoint] = React.useState<any>(null);
-  const [applications, setApplications] = React.useState<any>(null);
+  const [applications, setApplication] = React.useState<any>(null);
```

Comment:
Applications is plural, so should `setApplications` be. You've changed it to `setApplication`.